app/src/main/java/com/muzima/view/patients/PatientSummaryActivity.java

```diff
@@ -521,10 +521,10 @@ private String getFormattedPatientAddress(PersonAddress personAddress) {
     private String getObsByPatientUuidAndConceptId(String patientUuid, int conceptId) throws JSONException, ObservationController.LoadObservationException {
         try {
             List<Observation> observations = observationController.getObservationsByPatientuuidAndConceptId(patientUuid, conceptId);
-            Concept concept = conceptController.getConceptById(conceptId);
             Collections.sort(observations, observationDateTimeComparator);
             if (observations.size() > 0) {
                 Observation obs = observations.get(0);
+                Concept concept = obs.getConcept();
                 if (concept.isDatetime())
```

Comment:
You cant get .isDateTime from concept retrieved from obs, instead do this Concept concept = conceptController.getConceptById(conceptId);